lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -3985,6 +3985,79 @@ class ConvertAtenIndexSelectOp : public OpConversionPattern<AtenIndexSelectOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenIndexTensorOp : public OpConversionPattern<AtenIndexTensorOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenIndexTensorOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    if (failed(verifyLinalgCompatibleTypes(op, rewriter)))
+      return failure();
+
+    Location loc = op.getLoc();
+    Value input = adaptor.self();
+    Value indices = op.indices();
+    SmallVector<Value> indicesTuple;
+    if (!getListConstructElements(indices, indicesTuple)) {
+      return rewriter.notifyMatchFailure(
+          op, "unimplemented: the indices are not present.");
```

Comment:
nit: Here there can be an error even if the indices are present, for they could be constructed using something other than a PrimListConstruct. I would rewrite the error message as:

```
unimplemented: the indices list is not from a list construct
```